modules/bot/models/Chat.php

```diff
@@ -32,12 +35,6 @@ public function behaviors()
         ];
     }
 
-    public function getUsers()
-    {
-        return $this->hasMany(User::className(), ['id' => 'user_id'])
-                    ->viaTable('bot_chat_bot_user', ['chat_id' => 'id']);
-    }
-
```

Comment:
Follow [this comment](https://github.com/opensourcewebsite-org/opensourcewebsite-org/pull/286/files#r388206159)